aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -339,6 +347,12 @@ public class AwsProperties implements Serializable {
    */
   public static final String CLIENT_ASSUME_ROLE_REGION = "client.assume-role.region";
 
+  /**
+   * Used by {@link org.apache.iceberg.aws.AwsClientFactories.DefaultAwsClientFactory}. If set, all
+   * AWS clients except STS client will use * the given region instead of the default region chain.
+   */
+  public static final String AWS_REGION = "aws.region";
```

Comment:
in that case I think it's better to call it `client.region`?

Reply:
ok @jackye1995 I changed to `client.region`. Please check.